lib/Dancer2/Logger/Console/Colored.pm

```diff
@@ -172,11 +172,18 @@ __END__
 
 This is a logging engine that allows you to print colored debug messages on
 the standard error output. It is based on L<Dancer2::Logger::Console>. Refer
-to its documentation for how to configure the format of your log message.
+to L<Dancer::Logger::Abstract> and L<Dancer2::Config> for how to configure the format of your log message.
```

Comment:
The first one is for Dancer version 1. I think you meant Dancer2::Core::Role::Logger?

Reply:
Yes, thanks! I have updated the PR.